src/utils/props.js

```diff
@@ -51,28 +51,35 @@ export const PROPS_DISPLAY_OBJECT = {
  * Can be either texture or image
  *
  * @param {string} elementType
- * @param {PIXI.Texture|undefined} texture
- * @param {string|undefined} image
- * @param {boolean|undefined} crossorigin
- * @returns {PIXI.Texture}
+ * @param {object} props
```

Comment:
maybe you can specify props like I did in my other PR?
#125

Reply:
I intentionally removed them as these are all defined in the new TS definition which gives way better type support

Reply:
Even if you're not using typescript (type definitions are automatically picked up in VSCode and many other editors)

Reply:
You are right 👍